app/policies/user_policy.rb

```diff
@@ -37,6 +37,10 @@ def update?
     true
   end
 
+  def add_tag_category_tags?
+    user.hr_or_above?
```

Comment:
@harold93 I'm getting rid of all permissions that query access_level or access_level_int.
Can you please add a comment TODO UpdatePermission ?